src/plugin/nerd_plugin.js

```diff
@@ -254,7 +254,7 @@ class NerdPluginVirtual extends EventEmitter {
 
     const valid = yield this.balance.checkAndSettleOutgoing(transfer.amount)
     const validAmount = (typeof transfer.amount === 'string' &&
-      !isNaN(transfer.amount - 0) && (transfer.amount - 0 >= 0))
+      !isNaN(transfer.amount - 0) && (transfer.amount - 0 > 0))
```

Comment:
added a function in `util` that `JSON.parse`es a string and returns it only if the parse was successful. The functions where this is used already check to make sure that it isn't NaN, and if it is then they will emit an `InvalidFieldsError`.